src/OpenTelemetry.Exporter.OpenTelemetryProtocol/OtlpExporter.cs

```diff
@@ -37,7 +40,11 @@ public class OtlpExporter : BaseExporter<Activity>
         private const string DefaultServiceName = "OpenTelemetry Exporter";
 
         private readonly OtlpExporterOptions options;
+#if NETSTANDARD2_1
+        private readonly GrpcChannel channel;
+#else
         private readonly Channel channel;
+#endif
```

Comment:
Just a minor thing...You don't need to use the compiler directives here if you use ChannelBase instead of the specific Channel types.
```suggestion
        private readonly ChannelBase channel;
```

Reply:
We call `ShutdownAsync` on the channel here https://github.com/open-telemetry/opentelemetry-dotnet/blob/64777d3220da41e3573b32e7d83c39061c9b6dcd/src/OpenTelemetry.Exporter.OpenTelemetryProtocol/OtlpExporter.cs#L157

This is not a method on `ChannelBase`. Are you familiar with a different pattern for managing the lifetime of the channel?

Reply:
It looks like ShutdownAsync is defined on ChannelBase.
https://github.com/grpc/grpc/blob/master/src/csharp/Grpc.Core.Api/ChannelBase.cs#L53-L68

Reply:
Ah, yes ok I see what's going on. The net45, net46 and netstandard2.0 builds of the exporter depend on an older version (2.23.0) of the grpc library. `ShutdownAsync` was added after this version here https://github.com/grpc/grpc/commit/abeac187b6744302ff73ca20897f11c816cb3801#diff-218ffbfa6d10bb57abf89df29e76a18a0785a03c38952c219a84c08aada89545.

It might make sense to update the version, but I think makes sense as a separate concern from this PR